### PR TITLE
Added an `auto_mode` input parameter to `ElevenLabsTTSService`

### PR DESCRIPTION
**Pull request by markbackman**
#### Please describe the changes in your PR. If it is addressing an issue, please reference that as well.

- Made `auto_mode` and input parameter
- Made `eleven_flash_v2_5` the default model
- Updated the language logic to permit specific models to provide a language

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Set `auto_mode: True` for the `ElevenLabsTTSService`. This change
+  significantly reduces the latency by disabling the chunk schedule and all
+  buffers.
```
Comment (Vaibhav-Lodha) [inline CHANGELOG.md:47]: i agree to make this true by default but lets add it in input params to make it configurable if someone wants to exclude it

Comment (markbackman) [reply CHANGELOG.md:47]: Good point! I'll make that change in this PR.

**Comment by Vaibhav-Lodha** [comment]
They have also released `eleven_flash_v2_5`, and recommend using it over `eleven_turbo_v2_5`, maybe we can consider changing the default as well.
 
image

**Comment by markbackman** [comment]
> They have also released([notes](https://help.elevenlabs.io/hc/en-us/articles/17883183930129-What-models-do-you-offer-and-what-is-the-difference-between-them)) `eleven_flash_v2_5`, and recommend using it over `eleven_turbo_v2_5`, maybe we can consider changing the default as well.
> 
> image

Good call! Updated. Thanks @Vaibhav-Lodha 🙌

**Review by aconchillo — APPROVED** [review]

**Comment by aconchillo** [comment]
LGTM! Thanks!

**Pull request merged**